src/test/ledger/BookDirs_test.cpp

```diff
@@ -94,6 +96,7 @@ struct BookDirs_test : public beast::unit_test::suite
     void run() override
     {
         test_bookdir();
+        test_bookdir(featureFlow, featureToStrandV2);
```

Comment:
This one doesn't need `test_bookdir(featureFlow)`?  If not it might be a good idea to include a comment why.